cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This placeholder file should not be added

Reply:
> I took a quick look at this PR and tried running the accessibility tests. This looks promising, but I think there are things to improve. I wonder if it's really necessary to have so much differing code just to be able to support both CLI and GUI tests - ideally the same test code should work for both.
> 
> Runnin the Cypress tests now shows test failures because of accessibility problems in the current templates. This is to be expected, but I don't think we can merge a PR with failing tests because it would create chaos for development processes. So either the a11y issues should be fixed, or more likely, the failing tests should be marked as "skipped" or "expected to fail".
> 
> Minor housekeeping details: There were some extra, unnecessary files included in the PR. Many JS files also had unnecessary extra lines at the end.

Yes, all the "extra" has to be removed. I will do that before the next push. Also, JavaScript is not yet tested and before the merge, code checking also needs to be enabled as agreed.

Could we have a short design/review session together to go over what we actually want (reference: "ideally the same test code should work for both")?